crates/ruff_python_parser/tests/snapshots/invalid_syntax@try_stmt_missing_except_finally.py.snap

```diff
@@ -0,0 +1,55 @@
+---
+source: crates/ruff_python_parser/tests/fixtures.rs
+input_file: crates/ruff_python_parser/resources/inline/err/try_stmt_missing_except_finally.py
+---
+## AST
+
+```
+Module(
+    ModModule {
+        range: 0..19,
+        body: [
+            Try(
+                StmtTry {
+                    range: 0..4,
+                    body: [],
+                    handlers: [],
+                    orelse: [],
+                    finalbody: [],
+                    is_star: false,
+                },
+            ),
+            Try(
+                StmtTry {
+                    range: 5..18,
+                    body: [
+                        Pass(
+                            StmtPass {
+                                range: 14..18,
+                            },
+                        ),
+                    ],
+                    handlers: [],
+                    orelse: [],
+                    finalbody: [],
+                    is_star: false,
+                },
+            ),
+        ],
+    },
+)
+```
+## Errors
+
+  |
+1 | try:
+2 | try:
+  | ^^^ Syntax Error: Expected an indented block after `try` statement
```

Comment:
I don't think this is the intended error message. Any chance that the second `try:` clause is unintentional?

Reply:
Yeah, that was unintentional. Thanks!